public/app/features/explore/Wrapper.tsx

```diff
@@ -40,6 +53,17 @@ class WrapperUnconnected extends PureComponent<Props> {
 
     const richHistory = getRichHistory();
     this.props.richHistoryUpdatedAction({ richHistory });
+
+    const autoLoadLogsVolume = store.getBool(AUTO_LOAD_LOGS_VOLUME_SETTING_KEY, false);
+    this.props.storeAutoLoadLogsVolumeAction(autoLoadLogsVolume);
```

Comment:
i think we should do this in https://github.com/grafana/grafana/blob/main/public/app/features/explore/state/main.ts when we load the initial state, wdyt?

Reply:
You mean [line 172](https://github.com/grafana/grafana/blob/main/public/app/features/explore/state/main.ts#L172)? That would make reducer depend on local storage and it won't be pure anymore 🤔